src/OpenTelemetry/Trace/Configuration/OpenTelemetrySdk.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <copyright file="OpenTelemetrySdk.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using OpenTelemetry.Trace.Export;
+
+namespace OpenTelemetry.Trace.Configuration
+{
+    public class OpenTelemetrySdk
+    {
+        static OpenTelemetrySdk()
+        {
+            Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+            Activity.ForceDefaultIdFormat = true;
+        }
+
+        /// <summary>
+        /// Enables OpenTelemetry.
+        /// </summary>
+        /// <param name="configureOpenTelemetryBuilder">Function that configures OpenTelemetryBuilder.</param>
+        /// <remarks>
+        /// Basic implementation only. Most logic from TracerBuilder will be ported here.
+        /// </remarks>
+        public static void EnableOpenTelemetry(Action<OpenTelemetryBuilder> configureOpenTelemetryBuilder)
```

Comment:
Something feels a bit off about `OpenTelemetry.Trace.Configuration.OpenTelemetrySdk.EnableOpenTelemetry`. How would that look with both tracing and metrics? Would we need to do...
```csharp
OpenTelemetry.Trace.Configuration.OpenTelemetrySdk.EnableOpenTelemetry(...);
OpenTelemetry.Metrics.Configuration.OpenTelemetrySdk.EnableOpenTelemetry(...);
```
I'm thinking it should be more like...
```csharp
OpenTelemetry.OpenTelemetrySdk
   .EnableTracing(...)
   .EnableMetrics(...)
```
Doesn't need to be done as part of this PR.

Reply:
Not in this PR:
* We probably want to consider moving Configuration outside Trace/Metrics namespace.
* In Python it is called [opentelemetry.sdk](https://github.com/open-telemetry/opentelemetry-python/blob/e84faa4c7569e0fe4744c54adb4d006d091d9a48/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L30).

Reply:
Agree. Something simpler like OpenTelemetry.Configuration or similar.